department-of-veterans-affairs/va.gov-cms
# VACMS-2261_3602: Add resources panel to clp & create downloadable external file type

## Description
See #2261 #3602 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/100887025-7ea3f800-3482-11eb-9186-ca259afb4636.png)
![image](https://user-images.githubusercontent.com/2404547/100887061-8a8fba00-3482-11eb-887c-9287b086dbb8.png)
![image](https://user-images.githubusercontent.com/2404547/100887091-911e3180-3482-11eb-949a-1db45946253e.png)
![image](https://user-images.githubusercontent.com/2404547/100887114-98453f80-3482-11eb-9ce8-e4bab1bee5ac.png)
![image](https://user-images.githubusercontent.com/2404547/100887143-9f6c4d80-3482-11eb-975b-061df61c18ad.png)


## QA steps
- As an administrator, go to `/media/add/document_external` and  add an external pdf file, setting the node to published.
- On save, visually verify that the file type is populated.
- As an administrator, go to `node/add/campaign_landing_page#edit-group-downloadable-resources`
- Visually verify that the resources panel appears as indicated by the screenshots.
- Verify that Media resources -> Select files allows you to select the external file you added.
- Verify enabling panel and not entering data in fields does not save and displays form error.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
